api/model/api.go

```diff
@@ -1163,6 +1163,13 @@ type GraphQLConfig struct {
 
 	// Supergraph holds the configuration for a GraphQL federation supergraph.
 	Supergraph GraphQLSupergraphConfig `json:"supergraph,omitempty"`
+
+	// Introspection holds the configuration for GraphQL Introspection
+	Introspection GraphQLIntrospectionConfig `json:"introspection"`
```

Comment:
Is `Introspection` required field in GraphQLConfig? If not, we should add the `omitempty` tag to make it optional in CRD.

The same thing also applies to `Disabled` field in `GraphQLIntrospectionConfig`.